.circleci/build-debian.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+#
+# Build for Debian in a docker container
+#
+
+# bailout on errors and echo commands.
+set -xe
+
+DOCKER_SOCK="unix:///var/run/docker.sock"
+
+echo "DOCKER_OPTS=\"-H tcp://127.0.0.1:2375 -H $DOCKER_SOCK -s overlay2\"" | sudo tee /etc/default/docker > /dev/null
+sudo service docker restart
+sleep 5;
+
+if [ "$EMU" = "on" ]; then
+  if [ "$CONTAINER_DISTRO" = "raspbian" ]; then
+      docker run --rm --privileged multiarch/qemu-user-static:register --reset
+  else
+      docker run --rm --privileged --cap-add=ALL --security-opt="seccomp=unconfined" multiarch/qemu-user-static --reset --credential yes --persistent yes
+  fi
+fi
+
+WORK_DIR=$(pwd):/ci-source
+
+docker run --privileged --cap-add=ALL --security-opt="seccomp=unconfined" -d -ti -e "container=docker"  -v $WORK_DIR:rw $DOCKER_IMAGE /bin/bash
+DOCKER_CONTAINER_ID=$(docker ps --last 4 | grep $CONTAINER_DISTRO | awk '{print $1}')
+
+docker exec --privileged -ti $DOCKER_CONTAINER_ID apt-get update
+docker exec --privileged -ti $DOCKER_CONTAINER_ID apt-get -y install apt-transport-https wget curl gnupg2
+
+#docker exec --privileged -ti $DOCKER_CONTAINER_ID /bin/bash -xec \
+#  "wget -q 'https://dl.cloudsmith.io/public/bbn-projects/bbn-repo/cfg/gpg/gpg.070C975769B2A67A.key' -O- | apt-key add -"
+#docker exec --privileged -ti $DOCKER_CONTAINER_ID /bin/bash -xec \
+#  "wget -q 'https://dl.cloudsmith.io/public/bbn-projects/bbn-repo/cfg/setup/config.deb.txt?distro=${PKG_DISTRO}&codename=${PKG_RELEASE}' -O- | tee -a /etc/apt/sources.list"
+
+#docker exec --privileged -ti $DOCKER_CONTAINER_ID apt-get upgrade
+docker exec --privileged -ti $DOCKER_CONTAINER_ID apt-get -y install dpkg-dev devscripts equivs pkg-config apt-utils fakeroot
+docker exec --privileged -ti $DOCKER_CONTAINER_ID apt-get -y install autotools-dev autoconf dh-exec cmake gettext git-core \
+    libgtk-3-dev                           \
+    libgtk2.0-dev                          \
+    libgl1-mesa-dev                        \
+    libglu1-mesa-dev                       \
+    zlib1g-dev                             \
+    libjpeg-dev                            \
+    libpng-dev                             \
+    libtiff5-dev                           \
+    libsm-dev                              \
+    libexpat1-dev                          \
+    libxt-dev                              \
+    libcurl4-openssl-dev                   \
+    libgstreamer1.0-dev                    \
+    libgstreamer-plugins-base1.0-dev       \
+    libgstreamer-plugins-bad1.0-dev        \
+    libwebkit2gtk-4.0-dev                  \
+    libnotify-dev                          \
+    wget                                   \
+    doxygen                                \
+    graphviz                               \
+    meson                                  \
```

Comment:
We should need meson for building... nor cmake (installed above), or xsltproc.

It looks like many packages here are not really needed, it would be nice to restrict it to just the required ones, as it will make it simpler to update in the future.

Reply:
Feel free to remove unnecessary